packages/lesswrong/components/common/TabNavigationMenu/TabNavigationItem.tsx

```diff
@@ -97,7 +98,10 @@ const styles = (theme: ThemeType) => ({
   },
   navText: {
     ...theme.typography.body2,
-    color: theme.palette.text.bannerAdOverlay,
+    color: "inherit",
+    [isIfAnyoneBuildsItFrontPage]: {
```

Comment:
This should not be wrapped with the frontpage exception

Reply:
Moderate disagree.

Reply:
For this one I feel reasonably strongly it looks better (you most want to pay attention to it on the frontpage with a dynamic banner image, which we usually have).

Reply:
I ungated this, but I do still think it's worse, especially in light mode (which I'm not sure you intended ot change).